portfolio/src/main/java/com/google/sps/servlets/DeleteDataServlet.java

```diff
@@ -14,38 +14,62 @@
 
 package com.google.sps.servlets;
 
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Key;
 import com.google.appengine.api.datastore.Query;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Entity;
 import com.google.appengine.api.datastore.FetchOptions; 
-import com.google.appengine.api.datastore.FetchOptions.Builder; 
+import com.google.appengine.api.datastore.DatastoreServiceConfig;
+import com.google.appengine.api.datastore.DatastoreServiceConfig.Builder;  
+import com.google.appengine.api.datastore.ReadPolicy;
+import com.google.appengine.api.datastore.ReadPolicy.Consistency;
+import com.google.sps.data.Util; 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.List; 
 import java.util.ArrayList; 
+import java.util.Optional;
 
 /** Servlet responsible for deleting all quotes in Datastore. */
 @WebServlet("/deleteQuote")
 public class DeleteDataServlet extends HttpServlet {
 
   private static final String QUOTE = "Quote";
+  private static final String USER_EMAIL = "userEmail";
```

Comment:
Consider renaming this to,

PROPERTY_NAME_USER_EMAIL

Reply:
Updated.